app/javascript/app/pages/custom-compare/custom-compare-component.jsx

```diff
@@ -6,14 +6,55 @@ import cx from 'classnames';
 import Sticky from 'react-stickynode';
 import AnchorNav from 'components/anchor-nav';
 import BackButton from 'components/back-button';
+import Dropdown from 'components/dropdown';
+import NdcCompareAllTargetsProvider from 'providers/ndc-compare-all-targets-provider';
 
 import anchorNavRegularTheme from 'styles/themes/anchor-nav/anchor-nav-regular.scss';
+
 import layout from 'styles/layout.scss';
 import styles from './custom-compare-styles.scss';
 
-const CustomComparisonComponent = props => {
-  const { route, anchorLinks } = props;
+const COUNTRY_PLACEHOLDERS = [
+  'Add a first country',
+  'Add a second country',
+  'Add a third country'
+];
+
+const FiltersGroup = ({ data, countryPlaceholder, handleFilterChange }) => {
+  const {
+    countryParam,
+    countryValue,
+    contriesOptions,
+    documentParam,
+    documentValue,
+    documentOptions
+  } = data;
 
+  return (
+    <div className={styles.filter}>
+      <Dropdown
+        key={`${countryParam}-filter`}
+        className={styles.dropdown}
+        options={contriesOptions}
+        onValueChange={({ value }) => handleFilterChange(countryParam, value)}
+        value={countryValue}
+        placeholder={countryPlaceholder}
+        hideResetButton
```

Comment:
You could add the 'noAutoSort' prop to the dropdowns to prevent the sorting that now shows 'INDC, LTS, NDC' instead of 'INDC, NDC, LTS'

Reply:
Thanks, sure thing :+1: